.github/workflows/prestocpp-linux-adapters-build.yml

```diff
@@ -0,0 +1,31 @@
+name: prestocpp-linux-adapters-build
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - 'presto-native-execution/scripts/**'
+
+jobs:
+  prestocpp-linux-build-adapters:
+    runs-on: ubuntu-22.04
+    container:
+      image: prestodb/presto-native-dependency:0.290-20241014120930-e1fc090
+    steps:
+      - uses: actions/checkout@v4
+      - name: Fix git permissions
+        # Usually actions/checkout does this but as we run in a container
+        # it doesn't work
+        run: git config --global --add safe.directory ${GITHUB_WORKSPACE}
+      - name: Update submodules
+        run: |
+          cd presto-native-execution
+          make submodules
+      - name: Build all adapter dependencies
```

Comment:
We can remove this job. I am adding the adapters to the dependency image.
https://github.com/prestodb/presto/pull/24251

Reply:
This job is here so we can check if changes in scripts trigger a failure with the adapters. The triggers is only changes in the script dir.
I can change it to specifically only trigger on changes to setup-adapter.sh. We don't want to find out issues when building the image after having changed the setup-adapters. I think it is useful to have. It won't run very often.

Reply:
Actually, lets make it to run when we update velox or setup-adapters.sh.
The job is one of the quicker ones.

Reply:
But we don't need to install `/velox/scripts/setup-adapters.sh` since that is tested on the Velox CI.

Reply:
We also don't need this on a Velox update then.

Reply:
I was thinking if we could have a situation that isn't tested in Velox but might be used in PrestoC++. But I guess we'd need a full adapters build and PrestoC++ build. Which was what one CircleCI job originally did.